src/ring_theory/ideal/operations.lean

```diff
@@ -1604,3 +1611,97 @@ begin
 end
 
 end ring_hom
+
+namespace double_quot
+open ideal
+variables {R : Type u} [comm_ring R] (I J : ideal R)
+
+/-- define `quot_left_to_quot_sup` to be the obvious ring hom `R/I → R/(I ⊔ J)` -/
+def quot_left_to_quot_sup : I.quotient →+* (I ⊔ J).quotient :=
+  ideal.quotient.inclusion I (I ⊔ J) (le_sup_left)
+
+/-- This will be used to lift `quot_left_to_quot_sup` to a map `(R/I)/J' → R/(I ⊔ J)`-/
+lemma map_mk_le_ker_quot_left_to_quot_sum :
+  J.map (ideal.quotient.mk I) ≤ (quot_left_to_quot_sup I J).ker :=
```

Comment:
Is this true as follows?
```suggestion
  J.map (ideal.quotient.mk I) = (quot_left_to_quot_sup I J).ker :=
```

Reply:
Yes equality should hold !

Reply:
I'll have a go at it and push it as soon as I can (probably won't be able to until Monday though)

Reply:
Done !